### ISSUE TITLE
[Bug]: 使用同样的文本多次运行ernie-3.0系列模型进行文本特征抽取, 获得的特征不一致

### ISSUE THREAD

**Issue by miknyko** [PaddlePaddle/PaddleNLP#3944]
### 软件环境

```Markdown
- paddle-bfloat         0.1.7
- paddle2onnx           1.0.3
- paddlefsl             1.1.0
- paddlenlp             2.4.3
- paddlepaddle-gpu      2.3.2
```

### 重复问题

- [X] I have searched the existing issues

### 错误描述

```Markdown
你好, 我使用以下脚本, 多次运行, 每次获得的特征向量都不相同.
我使用其他模型, 如albert-chinese-tiny, 则没有这个问题.

但是, 如果我在一次进程里面, 连续抽取三次同样脚本, 结果是一致的.

只要进程关闭, 重启进程, 对同样的文本进行抽取, ernie-3.0-medium-zh或者mini就会给我不同的特征向量.

救救孩子
```

### 稳定复现步骤 & 代码

```
import paddle
from paddlenlp.transformers import *


model = AutoModel.from_pretrained('ernie-3.0-medium-zh')
tokenizer = AutoTokenizer.from_pretrained('ernie-3.0-medium-zh')
text = tokenizer('自然语言处理')
sequence_output, pooled_output = model(input_ids=paddle.to_tensor([text['input_ids']]))

# print(sequence_output[:, :, :10])
print(pooled_output[:, :10])


# 第一次运行结果
Tensor(shape=[1, 10], dtype=float32, place=Place(gpu:0), stop_gradient=False,
       [[ 0.20556203,  0.29300383, -0.22889188,  0.22407077, -0.08308320,
          0.66419160, -0.26418912,  0.85798335, -0.38627520,  0.58358848]])

# 第二次运行结果
Tensor(shape=[1, 10], dtype=float32, place=Place(gpu:0), stop_gradient=False,
       [[-0.26166221, -0.27211034,  0.53475565,  0.50548363,  0.38282761,
          0.36800560,  0.28447258, -0.41270202, -0.27387705,  0.57322991]])

**Comment by miknyko** [comment]
我猜测是不是通过这种方式调取ernie, 里面都会自带一个权重随机的classifier层?但是为什么同样的`AutoModel`用其他模型就没这个问题..是不是该统一一下?

**Comment by LiuChiachi** [comment]
model.eval()试试呢？是不是dropout没有关闭导致的

**Comment by miknyko** [comment]
eval()也是不行的

**Comment by LiuChiachi** [comment]
你好，看了一下pooler的参数是随机初始化的，预训练模型里面没有带
如果你要固定，可能需要在模型定义之前加上`paddle.seed(42)`

**Comment by miknyko** [comment]
好的明白了, 但是我比较疑问的是, 我进行文本embedding的话, 还有其他调用方式吗?我不太想要这个随机的pooling层, 里面的随机参数对文本特征抽取是不是意义不大

**Comment by LiuChiachi** [comment]
你可以操作`sequence_output`，没有经过Pool

**Comment by miknyko** [comment]
好的, 明白了, 谢谢
